cli/pom.xml

```diff
@@ -63,7 +63,7 @@
         <dependency>
             <groupId>me.tongfei</groupId>
             <artifactId>progressbar</artifactId>
-            <version>0.9.0</version>
+            <version>0.7.4</version>
```

Comment:
why Upgrading 0.9.0  to 0.7.4 ？

Reply:
Emmm.... It may be the mistake of dependabot.  Let me close this PR?